packages/nc-gui/components/smartsheet/sidebar/index.vue

```diff
@@ -21,11 +22,11 @@ const meta = inject(MetaInj, ref())
 
 const activeView = inject(ActiveViewInj, ref())
 
-const { activeTab } = useTabs()
+const { activeTab } = storeToRefs(useTabs())
 
 const { views, loadViews, isLoading } = useViews(meta)
```

Comment:
should apply `storeToRefs` for views & isLoading

Reply:
Both `loadViews, isLoading` are loading from `useViews` which is not a Pinia store